docs/wallets/ledger_desktop_wallet.md

```diff
@@ -50,33 +52,56 @@ After the app is set, please conducting the following steps to create a wallet w
 
     - Input your wallet name of your choice;
     - Tick "*Want to create with hardware wallet*" and choose "*Ledger*" as the wallet type. 
-    - For the network, three options of "**Testnet**", "**Mainnet**", and "**Custom Devnet**" are given. **Testnet** is Croeseid testnet which is our Cosmos-based testnet that has been running since last October. **Mainnet** is refers to Crypto.org Chain mainnet. Other than these, you may fill up the custom configuration to your own network.
+    - For the network, three options of "**Testnet Croeseid 4**", "**Mainnet**", and "**Custom Devnet**" are given. **Testnet Croeseid 4** is Croeseid testnet which is our Cosmos-based testnet that has been running since last October. **Mainnet** is refers to Crypto.org Chain mainnet. Other than these, you may fill up the custom configuration to your own network.
```

Comment:
```suggestion
For the network, three options of "**Testnet Croeseid 4**", "**Mainnet**", and "**Custom Devnet**" are given. **Testnet Croeseid 4** is Croeseid testnet which is our Cosmos-based testnet that has been running since October 2020. **Mainnet** refers to Crypto.org Chain mainnet. Other than these, you can also fill up the custom configuration to your own network.
```